app/_src/policies/meshtcproute.md

```diff
@@ -18,13 +18,35 @@ depending on where the request is coming from and where it's going to.
 
 ## TargetRef support matrix
 
+{% if_version gte:2.6.x %}
+{% tabs targetRef useUrlFragment=false %}
+{% tab targetRef Sidecar %}
```

Comment:
taken from https://github.com/kumahq/kuma/blob/d337897012afe45dc525dfd42fabcf561e9be923/pkg/plugins/policies/meshtcproute/api/v1alpha1/validator.go#L20-L50 - looks identical to previous one